composables/autoTeleport/useAutoTeleportTransactionActions.ts

```diff
@@ -58,9 +35,9 @@ export default function (actions: ComputedRef<AutoTeleportAction[]>) {
 
   watch(
     actions,
-    (newActions, prevActions) => {
-      newActions.forEach((action, index) => {
-        const cancelled = isCancelled(action, prevActions?.[index])
+    (actions) => {
```

Comment:
Since we are not using the previous state, can we use `watchEffect` instead of `watch` here?

Reply:
just checked, `watchEffect` does not watch for `deep` changes